### PR TITLE
6. Images and 12. Tables - adding consideration for role="none"

### LINKED ISSUE #211: 6. Images - add consideration for role="none"

### PR DESCRIPTION
**Pull request by drewnielson**
in addition to current references to role="presentation". Closes #211

### CODE REVIEW

File: docs/12DataTables.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Note: Linearization of table content is the presentation of a table’s two-dime
...
     -   **NOTE**: according to HTML5 specifications, the &lt;table&gt; element and associated elements and attributes may not be used in combination with ARIA role="table" and associated roles and attributes. If a table combines these techniques, it fails this test. ARIA role="grid", on the other hand, **may** be used on a &lt;table&gt; element.
 > For example, if an author applies the grid role to an HTML table element, the author does not need to apply the row and gridcell roles to the descendant HTML &lt;tr&gt; and &lt;td&gt; elements because the user agent will automatically make the appropriate translations. When the author is reusing a native host language table element and needs a gridcell element to span multiple rows or columns, the author SHOULD apply the appropriate host language attributes instead of WAI-ARIA aria-rowspan or aria-colspan properties (from [Accessible Rich Internet Applications (WAI-ARIA) 1.1](https://www.w3.org/TR/wai-aria-1.1/#grid)).
-1.  Check that an ARIA role="presentation" is NOT assigned to a data &lt;table&gt; element.
+1.  Check that an ARIA role is NOT set to "presentation" nor to "none" for a data &lt;table&gt; element.
```
Comment (kengdoj) [inline docs/12DataTables.md:34]: 1. Check that the data &lt;table&gt; element does not have role="presentation" or role="none".

File: docs/12DataTables.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ Note: Linearization of table content is the presentation of a table’s two-dime
 #### Layout Tables
 1.  Check that tables used purely for layout purposes:
     1.  Do NOT designate the layout as a table using ARIA role="table" and associated ARIA table attributes.
-    2.  If the table does not have role="presentation", do NOT include table structure and relationship elements and/or associated attributes (i.e., &lt;th&gt;, summary= (HTML4), &lt;caption&gt;, scope=, and/or headers=).
+    2.  If the table does not have role="presentation" nor role="none", do NOT include table structure and relationship elements and/or associated attributes (i.e., &lt;th&gt;, summary= (HTML4), &lt;caption&gt;, scope=, and/or headers=).
```
[the remark this answers is not in the document]
Comment (drewnielson) [reply docs/12DataTables.md:52]: Yep, I like it. Agreed that the original wording was awkward. Incorporated as suggested.

File: docs/12DataTables.md (hunk 1 above)
Comment (drewnielson) [reply docs/12DataTables.md:34]: fixed as suggested

**Comment by drewnielson** [comment]
Made changes to incorporate suggested edits. @cliffbert and @kengdoj please re-review.

**Pull request merged**